sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/webkey/JsonWebKey.java

```diff
@@ -200,7 +201,7 @@ public JsonWebKey setKeyType(KeyType keyType) {
      */
     @JsonProperty("key_ops")
     public List<KeyOperation> getKeyOps() {
-        return this.keyOps;
+        return Collections.unmodifiableList(this.keyOps);
```

Comment:
You need to document in the JavaDoc whenever an unmodifiable collection is returned.

Reply:
added the documentation